plugins/channelrx/sigmffilesink/sigmffilesinkbaseband.cpp

```diff
@@ -34,7 +34,6 @@ SigMFFileSinkBaseband::SigMFFileSinkBaseband() :
     m_specMax(0),
     m_squelchLevel(0),
     m_squelchOpen(false),
```

Comment:
Extra comma here

Reply:
Thx, fidex

Reply:
But the `../logging/logger.h:13:10: fatal error: QRecursiveMutex: No such file or directory` is probably issue with `Qt 5.12` right?

Reply:
> Note that the main thing holding us back to Ubuntu 20.04 and therefore Qt 5.12.8 is the Appveyor image for CI. However it seems possible to use Qt 5.15.2 inside this image but this needs some investigation and it is not ready right now.

What do you suggest, I can hold till the Qt version issue is resolved. Should I close this MR for now?

Reply:
Indeed we will not try to make it work for Qt 5.12 and also later versions with #define(s). So in any case it should wait for Qt upgrade whether it remains open and unmerged or closed and submitted later. I know keeping Qt 5.12 compatibility is a pain (another PR is failing to compile because of this) and upgrade should take place one way or the other and the sooner the better.

Reply:
https://github.com/f4exb/sdrangel/issues/1416 opened

Reply:
Now building for Ubuntu 22.04 with Qt 5.15